services/checks/checks_test.go

```diff
@@ -156,6 +163,44 @@ uEF33ScMPYpvHvBKv8+yBkJ9k4+DCfV4nDs6kKYwGhalvkkqwWkyfJffO+KW7a1m3y42WHpOnzBxLJ+I
 	})
 }
 
+func TestStartChecks(t *testing.T) {
+	t.Run("stt disabled", func(t *testing.T) {
+		sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+		db := reform.NewDB(sqlDB, postgresql.Dialect, nil)
+
+		defer func() {
+			require.NoError(t, sqlDB.Close())
+		}()
```

Comment:
I guess function wrapper can be omitted?

Reply:
No, it can't because `A deferred function's arguments are evaluated when the defer statement is evaluated.` https://blog.golang.org/defer-panic-and-recover

Here's an example: https://play.golang.org/p/5uZnIPOt7VJ